Templates/CSharp/Base/CollectionRequestBuilder.Base.template.tt

```diff
@@ -119,6 +119,20 @@ public string GetCollectionReferencesRequestBuilder(OdcmProperty odcmProperty)
 // Creates the indexer property definition for a standard navigation collection
 public string GetCollectionIndexRequestBuilder(OdcmProperty odcmProperty)
 {
+    // In case there is an action or function with the name of item. This is to avoid CS0102
+    // https://msdn.microsoft.com/en-US/library/26f2y168%28v=VS.90%29.aspx
+    var allMethods = new List<OdcmMethod>();
+    foreach (var method in odcmProperty.Projection.Type.AsOdcmClass().Methods)
+    {
+        allMethods.Add(method);
+        allMethods.AddRange(method.Overloads);
+    }
+    var renameItem = "";
+    if (allMethods.Exists(x => x.Name.Equals("item")))
```

Comment:
Does case matter for the Name? Don't assume that the case in the metadata is set in stone.